docs/config-file-curations-yml.md

```diff
@@ -46,6 +46,9 @@ These curations can be configured in a YAML file that is passed to the _analyzer
 amends the metadata provided by the packages themselves. This way, it is possible to fix broken VCS URLs or provide the
 location of source artifacts.
 
+Hint: If the `concluded_license` _and_ the `authors` are curated, this package will be skipped during the `scan` step,
+as no more information from the scanner is required.
```

Comment:
However, this is only true if [skipConcluded](https://github.com/oss-review-toolkit/ort/blob/01c598a04c5745dc7408f6f84b619bc62d13592e/model/src/main/resources/reference.conf#L100) is enabled.